server/controllers/atomizer-amqp.js

```diff
@@ -1,18 +1,22 @@
 // Access the callback-based API
 var amqp = require('amqplib/callback_api');
 var amqpConn = null;
-var exchange = 'atomizer';
 var atomizerChannel = null;
-exports.KEY_PI_REQUEST = 'atomizer.montecarlopi.request';
-const keys = [this.KEY_PI_REQUEST, 'atomizer.montecarlopi.result'];
+var exchange = process.env.EXCHANGE || 'atomizer';
+var topic = process.env.TOPIC || 'electrons';
+
+//Setup Event Emmiter
+var events = require('events');
+var em = new events.EventEmitter();
+
 
 exports.start = function () {
     //Create Connection
     //Use Localhost if connecting outside of a container, 
     //else use the container name [some-rabbit] for container to container communication 
-    amqp.connect('amqp://some-rabbit', function (error1, conn) {
+    amqp.connect('amqp://localhost', function (error1, conn) {
```

Comment:
Will this hard coded values cause issues in a docker container? Can we set this up to be a container ENV variable instead for the connection string?

Reply:
Yes,  this one can easily be replaced with a container ENV variable.  I should be able to just use the CONNECTIONSTRING that we are already using for the atomizer container.  I get that coded in

Reply:
I modified this value to use an ENV variable called RABBIT_NAME.